api/app.test.js

```diff
@@ -114,4 +114,27 @@ describe("GET /entry/:entryId", () => {
       })
     );
   });
+
+  describe("DELETE / entry", () => {
+    it("deletes the selected entry from the UI and database", async () => {
+      const startTime = new Date();
+      const endTime = new Date();
+      const data = await CalendarEntry.create({
+        eventId: "123",
+        creatorId: "456",
+        title: "Happy day",
+        description: "and a happy night too",
+        isAllDay: false,
+        startTimeUtc: startTime,
+        endTimeUtc: endTime,
+      });
```

Comment:
I thought we put this in a `before` since it's test setup?

Reply:
We don't need to, it felt clear to put it as the first step of the test here.

Reply:
I think it would be nice organizationally, since it's not part of the assertions.